cmd/oras/internal/display/status/track/target.go

```diff
@@ -35,6 +35,7 @@ type GraphTarget interface {
 
 type graphTarget struct {
 	oras.GraphTarget
+	tty          *os.File
```

Comment:
Manager should be the only way to access tty, trackable readers or targets should not access tty directly.

Reply:
The weirdness here is because this PR is part of https://github.com/oras-project/oras/pull/1474

Now that the console interface has merged, maybe the older PR is small enough